src/test/java/com/softserve/edu/greencity/ui/pages/econews/EconewsPage.java

```diff
@@ -50,20 +49,17 @@ private void visualiseElements() {
 	}
 
 	private void initElements() {
-		tagsComponent = new TagsComponent(driver);
-		createNewsButton = driver.findElement(By.id("create-button"));
-		itemsContainer = new ItemsContainer(driver);
 		gridView = driver.findElement(By.cssSelector("div[class*='gallery-view']"));
 		listView = driver.findElement(By.cssSelector("div[class*='list-view']"));
-		foundItems = driver.findElement(By.xpath("//p[@class=\"ng-star-inserted\"]"));
+		foundItems = driver.findElement(By.xpath("//p[@class='ng-star-inserted']"));
```

Comment:
please use By.className in such cases

Reply:
Fixed to foundItems = driver.findElement(By.cssSelector("p[class*='ng-star-inserted']"));